packages/cli/src/commands/publish/index.ts

```diff
@@ -12,6 +12,10 @@ function logReleases(pkgs: Array<{ name: string; newVersion: string }>) {
   log(mappedPkgs);
 }
 
+function logReleasesInJSON(pkgs: Array<{ name: string; newVersion: string }>) {
+  log(JSON.stringify(pkgs));
```

Comment:
`log` comes with an emoji prefix and stuff. Since this output is meant to be machine-parseable I would say that it's best to just call `console.log` directly here:
```suggestion
  console.log(JSON.stringify(pkgs));
```

Reply:
addressed